### ISSUE TITLE
Support onClick funtion on Link component

### ISSUE THREAD

**Issue by Alex-Ferreli** [molefrog/wouter#8]
Hi @molefrog, nice plugin, i love the Hooks approace!

I have discovered that the Link component didn't accept a onClick prop to do something other then the normal behavior of navigate.

I have implemented the code to support this functionality locally, can i make a PR to merge with your package?

Thanks in advance!

**Comment by molefrog** [comment]
@Alex-Ferreli Sounds great, sure send the PR, would be glad to review it. And thanks for the kind words!
Do I get it right that what you're trying to achieve is to be able to attach an event handler, which will be fired after the transition happens? Sort of:

```js
const afterNavigate = () => { console.log('navigated to /foo') }

<Link onClick={afterNavigate} href="/foo"><a>Click</a></Link>
```

**Comment by molefrog** [comment]
Closed via #12
